napari_clusters_plotter/_plotter.py

```diff
@@ -124,7 +124,7 @@ def manual_clustering_method(inside):
                 plot_cluster_name=clustering_ID,
             )
             if isinstance(self.analysed_layer, Labels):
-                self.layer_select.value.opacity = 0
+                self.layer_select.opacity = 0.2
```

Comment:
I tested a bit. I would prefer 0.25 or even 0.3

Reply:
I would like to not increase it too much so maybe we can stick to 0.25, so that the experience does not change too dramatically?

Reply:
0.25 would be fine for. (I could also live with 0.2). The slight increase was just a subjective suggestion.